modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1756,37 +1756,39 @@ export class Wallet implements IWallet {
    * @returns {*}
    */
   async send(params: SendOptions = {}): Promise<any> {
-    common.validateParams(params, ['address'], ['message', 'data']);
-
-    if (_.isUndefined(params.amount)) {
-      throw new Error('missing required parameter amount');
-    }
+    const coin = this.baseCoin;
+    if (params.type == 'transfer') {
```

Comment:
check if address and amount exist just for `transfer`

Reply:
This seems like a pretty risky change since other types of transactions may have address amount, like staking.

Maybe we should support zero amounts for create token account transactions